packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -416,14 +416,14 @@ export interface IEnqueueSummarizeOptions extends IOnDemandSummarizeOptions {
     readonly override?: boolean;
 }
 
-// @alpha (undocumented)
+// @alpha
 export interface IGCMetadata {
     readonly gcFeature?: GCVersion;
     readonly gcFeatureMatrix?: GCFeatureMatrix;
     readonly sessionExpiryTimeoutMs?: number;
     // @deprecated
     readonly sweepEnabled?: boolean;
-    readonly sweepTimeoutMs?: number;
+    readonly tombstoneTimeoutMs?: number;
```

Comment:
Are we going to also store the `sweepGracePeriodMs` here? From a very quick glance it looks like there is no sweep timeout.

Reply:
Interesting point about how it's surprising to not see sweep timeout persisted.

`sweepGracePeriodMs` is NOT persisted, by design.

Reply:
Adding a comment.